framework/generated/vulkan_generators/vulkan_api_call_encoders_body_generator.py

```diff
@@ -233,7 +233,11 @@ def make_cmd_body(self, return_type, name, values):
         if name == "vkCreateInstance" or name == "vkQueuePresentKHR":
             body += indent + 'auto api_call_lock = VulkanCaptureManager::AcquireExclusiveApiCallLock();\n'
         else:
-            body += indent + 'auto api_call_lock = VulkanCaptureManager::AcquireSharedApiCallLock();\n'
+            body += indent + 'auto enable_exclusive_lock = VulkanCaptureManager::Get()->GetEnableExclusiveLock();\n'
+            body += indent + 'std::shared_lock<CaptureManager::ApiCallMutexT> shared_api_call_lock;\n'
+            body += indent + 'std::unique_lock<CaptureManager::ApiCallMutexT> exclusive_api_call_lock;\n'
+            body += indent + 'if (enable_exclusive_lock) exclusive_api_call_lock = VulkanCaptureManager::AcquireExclusiveApiCallLock();\n'
+            body += indent + 'else shared_api_call_lock = VulkanCaptureManager::AcquireSharedApiCallLock();\n'
```

Comment:
I know this is generated but shouldn't it follow the coding guidelines a bit and use curly braces and spread over eight lines?
At least putting the branch bodies on their own lines would let a person set a breakpoint on them.

Reply:
fixed it. Thank you.